subprojects/packagefiles/theora/meson.build

```diff
@@ -0,0 +1,269 @@
+project(
+  'theora',
+  'c',
+  'cpp',
+  version: '1.1.1',
+  meson_version: '>= 0.59.0',
+  license: 'BSD-3-Clause',
+  default_options: ['c_std=c11'],
+)
+
+conf = configuration_data()
+
+enable_valgrind = get_option('valgrind-testing').allowed()
+
+test_env = []
+
+if enable_valgrind
+  valgrind_exe = find_program('valgrind')
+
+  conf.set('HAVE_VALGRIND', valgrind_exe.found())
+
+  if conf.get('HAVE_VALGRIND')
+    test_env += [
+      '-q',
+      '--leak-check=yes',
+      '--show-reachable=yes',
+      '--num-callers=100',
+    ]
+
+    message(
+      'Tests will be run under: @0@'.format(env.get('VALGRIND_ENVIRONMENT')),
+    )
+  else
+    message('Valgrind not found')
```

Comment:
Redundant, Meson logs whether or not it's found.

Reply:
Ported from https://gitlab.xiph.org/xiph/theora/-/blob/master/configure.ac#L166, but I'll remove it.